src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  @include smooth;
+
+  border: solid 1px #f3f3f3;
+  width: $card-width;
+  height: $card-height;
+
+  background-image: url($prod-img-source);
+  background-position-y: 32px;
+  background-position-x: 19px;
+  background-size: $prod-img-size;
+  background-repeat: no-repeat;
+
+  &__text {
+    @include smooth;
+
+    line-height: 18px;
+    font-size: $font-size-default;
+    font-weight: 500;
+    color: #060b35;
+
+    margin-top: 206px;
+    padding: 0 16px;
+  }
+
+  &__prod-code {
+    color: #616070;
+    font-size: 10px;
+    padding: 4px 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    margin-top: 16px;
+    padding: 0 16px;
+
+    &-text {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+
+      margin-left: 17px;
+    }
+  }
+
+  &__price {
+    font-size: $font-size-default;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+
+    margin-top: 24px;
+    padding: 0 16px;
+    display: flex;
+    flex-direction: row;
+
+    &-value {
+      margin-left: 85px;
+      font-weight: 700;
+      font-size: 16px;
+      color: #000;
+    }
+  }
+
+  &__button-buy {
+    @include smooth;
+
+    width: $button-width;
+    height: $button-height;
+
+    border-width: 0;
+    border-radius: 5px;
+    background-color: $button-color;
+    margin: 16px;
+
+    color: $button-font-color;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+
+    &:hover {
+      background-color: $button-color--hover;
+      color: $button-font-color--hover;
+      border: solid 1px #00acdc;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
Scaling the card on hover (`transform: scale(1.2)`) might affect the layout, especially if the cards are in a grid or flex layout. Consider the impact on surrounding elements and ensure it doesn't cause layout issues.